packages/app/src/components/catalog/EntityPage.tsx

```diff
@@ -286,7 +286,7 @@ const ServiceEntityPage = ({ entity }: { entity: Entity }) => (
     <EntityPageLayout.Content
       path="/docs/*"
       title="Docs"
-      element={<DocsRouter entity={entity} />}
+      element={<DocsRouter />}
```

Comment:
Based on https://backstage.io/docs/plugins/composability#porting-existing-plugins, should this be called `EntityDocsContent`?

Reply:
hmmm yes I believe it should use the  routable extension  https://github.com/backstage/backstage/blob/master/plugins/techdocs/src/plugin.ts#L108 instead?

Reply:
And maybe sub routes refs also play a role here, if you have a router inside your routable component.

Reply:
@Fox32 @emmaindal it seems these changes would be part of a refactor of the catalog in the main app and outside scope of this PR?

Reply:
EntityTechdocsContent is the routable extension exported by the Techdocs while DocsRouter is the plain router component. It looks like to use the routable extension in the app, EntityPage.tsx has to be ported from using EntityPageLayout to EntityLayout. I think that would be under the scope of porting the main app and outside this PR's scope of porting the Techdocs frontend plugin.